### PR TITLE
Update Copilot docs - GitHub Copilot is now free for all teachers verified on GitHub Global Campus.

### LINKED ISSUE #20500: GitHub Copilot is now free for teachers
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

- [About billing for GitHub Copilot](https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot#about-billing-for-github-copilot)
- [Getting started with GitHub Copilot in Visual Studio Code](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio-code)
- [Getting started with GitHub Copilot in Visual Studio](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio)
- [Getting started with GitHub Copilot in a JetBrains IDE](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide)
- [Getting started with GitHub Copilot in Neovim](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-neovim)

Likely many more

### What part(s) of the article would you like to see updated?

GitHub Copilot is now free for teachers, most of the docs say "Verified students and maintainers of popular open source projects on GitHub are eligible to use GitHub Copilot for free" or "GitHub Copilot is free to use for verified students and maintainers of popular open source projects." The copy should be updated to reflect this excellent new change.

### Additional information

- [Please provide free license of GitHub Copilot to teachers as well #92](https://github.com/community/Global-Campus-Teachers/discussions/92#discussioncomment-3598302)
- [GitHub Copilot is now free for teachers #118](https://github.com/community/Global-Campus-Teachers/discussions/118?mc_cid=b9e94c38ce&mc_eid=18aefff4d8#discussion-4375925)
- [Copilot is now available free for verified teachers](https://github.blog/2022-09-08-github-copilot-now-available-for-teachers/)

### PR DESCRIPTION
**Pull request by luc122c**
### Why:
[GitHub Copilot](https://github.us11.list-manage.com/track/click?u=9d7ced8c4bbd6c2f238673f0f&id=285d96bfcd&e=18aefff4d8) is now free for all teachers verified on [GitHub Global Campus](https://github.us11.list-manage.com/track/click?u=9d7ced8c4bbd6c2f238673f0f&id=92482c139c&e=18aefff4d8). This PR updates the docs to reference this policy change. Closes #20500

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Before: 
image
After:
image

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by luc122c** [comment]
I think I've found and updated all the references. You may want to add additional info in to some sections such as [GitHub Copilot pricing](https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#github-copilot-pricing).

**Review by nuhmanpk — COMMENTED** [review]
👍🏻

**Comment by cmwilson21** [comment]
@luc122c Thanks for opening a PR and linking it to your issue! ✨ 

I'll get this triaged for review!

File: content/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ Before starting a paid subscription, you can set up a one-time 60-day trial to e
...
+A free subscription for {% data variables.product.prodname_copilot %} is available to verified students, teachers and maintainers of popular open-source repositories on {% data variables.product.company_short %}. If you meet the criteria as an open source maintainer, you will be automatically notified when you visit the {% data variables.product.prodname_copilot %} subscription page. As a student, if you currently receive the {% data variables.product.prodname_student_pack %}, you will also be offered a free subscription when you visit the {% data variables.product.prodname_copilot %} subscription page. For more information about the {% data variables.product.prodname_student_pack %}, see "[Apply to {% data variables.product.prodname_global_campus %} as a student](/free-pro-team@latest/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)."
```
Comment (am-stead) [inline content/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot.md:20]:
```suggestion
A free subscription for {% data variables.product.prodname_copilot %} is available to verified students, teachers, and maintainers of popular open-source repositories on {% data variables.product.company_short %}. If you meet the criteria as an open source maintainer, you will be automatically notified when you visit the {% data variables.product.prodname_copilot %} subscription page. As a student, if you currently receive the {% data variables.product.prodname_student_pack %}, you will also be offered a free subscription when you visit the {% data variables.product.prodname_copilot %} subscription page. For more information about the {% data variables.product.prodname_student_pack %}, see "[Apply to {% data variables.product.prodname_global_campus %} as a student](/free-pro-team@latest/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)."
```

File: content/copilot/overview-of-github-copilot/about-github-copilot.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ You are responsible for ensuring the security and quality of your code. We recom
...
+{% data variables.product.prodname_copilot %} is a paid feature, requiring a monthly or yearly subscription. Verified students, teachers and maintainers of popular open source projects on {% data variables.product.prodname_dotcom %} are eligible to use {% data variables.product.prodname_copilot %} for free. If you meet the criteria for a free {% data variables.product.prodname_copilot %} subscription, you will be automatically notified when you visit the {% data variables.product.prodname_copilot %} subscription page. If you do not meet the criteria for a free {% data variables.product.prodname_copilot %} subscription, you will be offered a 60 day free trial, after which a paid subscription is required for continued use. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
```
Comment (am-stead) [inline content/copilot/overview-of-github-copilot/about-github-copilot.md:31]:
```suggestion
{% data variables.product.prodname_copilot %} is a paid feature, requiring a monthly or yearly subscription. Verified students, teachers, and maintainers of popular open source projects on {% data variables.product.prodname_dotcom %} are eligible to use {% data variables.product.prodname_copilot %} for free. If you meet the criteria for a free {% data variables.product.prodname_copilot %} subscription, you will be automatically notified when you visit the {% data variables.product.prodname_copilot %} subscription page. If you do not meet the criteria for a free {% data variables.product.prodname_copilot %} subscription, you will be offered a 60 day free trial, after which a paid subscription is required for continued use. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
```

File: data/reusables/copilot/copilot-prerequisites.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-- {% data variables.product.prodname_copilot %} is free to use for verified students and open source maintainers. 
-- If you are not a student or open source maintainer, you can try {% data variables.product.prodname_copilot %} for free with a one-time 60 day trial. After the free trial, you will need a paid subscription for continued use. You must provide billing information in order to start a free trial. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
+- {% data variables.product.prodname_copilot %} is free to use for verified students, teachers and open source maintainers.
```
Comment (am-stead) [inline data/reusables/copilot/copilot-prerequisites.md:1]:
```suggestion
- {% data variables.product.prodname_copilot %} is free to use for verified students, teachers, and open source maintainers. 
```

File: data/reusables/copilot/copilot-prerequisites.md [hunk 4]
```diff
@@ -1,2 +1,2 @@
-- {% data variables.product.prodname_copilot %} is free to use for verified students and open source maintainers. 
-- If you are not a student or open source maintainer, you can try {% data variables.product.prodname_copilot %} for free with a one-time 60 day trial. After the free trial, you will need a paid subscription for continued use. You must provide billing information in order to start a free trial. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
+- {% data variables.product.prodname_copilot %} is free to use for verified students, teachers and open source maintainers. 
+- If you are not a student, teacher or open source maintainer, you can try {% data variables.product.prodname_copilot %} for free with a one-time 60 day trial. After the free trial, you will need a paid subscription for continued use. You must provide billing information in order to start a free trial. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
```
Comment (am-stead) [inline data/reusables/copilot/copilot-prerequisites.md:2]:
```suggestion
- If you are not a student, teacher, or open source maintainer, you can try {% data variables.product.prodname_copilot %} for free with a one-time 60 day trial. After the free trial, you will need a paid subscription for continued use. You must provide billing information in order to start a free trial. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
```

File: data/reusables/gated-features/copilot.md [hunk 5]
```diff
@@ -1 +1 @@
...
+{% data variables.product.prodname_copilot %} is available to {% data variables.product.company_short %} customers with a personal account on {% data variables.product.prodname_dotcom_the_website %}. {% data variables.product.prodname_copilot %} is free to use for verified students, teachers and maintainers of popular open source projects. If you are not a student, teacher or maintainer of a popular open source project, you can try {% data variables.product.prodname_copilot %} for free with a one-time 60 day trial. After the free trial, you will need a paid subscription for continued use. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
```
Comment (am-stead) [inline data/reusables/gated-features/copilot.md:1]:
```suggestion
{% data variables.product.prodname_copilot %} is available to {% data variables.product.company_short %} customers with a personal account on {% data variables.product.prodname_dotcom_the_website %}. {% data variables.product.prodname_copilot %} is free to use for verified students, teachers, and maintainers of popular open source projects. If you are not a student, teacher, or maintainer of a popular open source project, you can try {% data variables.product.prodname_copilot %} for free with a one-time 60 day trial. After the free trial, you will need a paid subscription for continued use. For more information, see "[About billing for {% data variables.product.prodname_copilot %}](/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)."
```

**Review by am-stead — COMMENTED** [review]
Thank you very much for creating a PR! The changes have been approved and they will be merged soon.

**Review by cmwilson21 — APPROVED** [review]

**Pull request merged**